src/tools/dart-tool.md

```diff
@@ -47,26 +47,25 @@ The following table shows which commands you can use with the `dart` tool.
 |---------+--------------------------------+-----------------------------------|
 | Command | Example of use                 | More information                  |
 |---------|--------------------------------|-----------------------------------|
-| analyze | `dart analyze`     | Analyzes the project's Dart source code.<br>Use instead of [`dartanalyzer`][]. |
-| compile | `dart compile exe <DART_FILE>` | Compiles Dart to various formats.<br>Use instead of [`dart2js`][] and [`dart2native`][]. | 
-| create  | `dart create <DIRECTORY>`      | Creates a new project.<br>Use instead of [`stagehand`.][] | 
-| format  | `dart format <DIRECTORY|DART_FILE>` | Formats Dart source code.<br>Use instead of [`dartfmt`][]. | 
-| pub     | `dart pub <PUB_COMMAND>`       | Works with packages.<br>Use instead of [`pub`][]. | 
-| run     | `dart run <DART_FILE>`         | Runs a Dart program. <br>Use instead of the pre-existing [Dart VM command][dart-vm]. | 
-| test    | `dart test [<DIRECTORY|DART_FILE>]` | Runs tests in this package.<br>Use instead of [`pub run test`][`pub`]. |
+| `analyze` | `dart analyze`     | Analyzes the project's Dart source code.<br>Use instead of `dartanalyzer`.<br>[Learn more.][analyze] |
```

Comment:
@devoncarew should there be a row here for `dart fix`?

Reply:
Yep, that's in issue #2858, and I should at least mention it in this table. I still need content for real `dart fix` docs, though...

Reply:
Is the syntax exactly the same as what's documented in https://pub.dev/packages/dartfix — just add a space?

Reply:
No, it's a different tool (with a similar focus). Here's what the CLI reports:

```
Fix Dart source code.

This tool looks for and fixes analysis issues that have associated automated fixes or issues that have associated package API migration
information. See dart.dev/go/dart-fix for more information about how automated package API changes work.

To use the tool, run either 'dart fix --dry-run' for a preview of the proposed changes for a project, or 'dart fix --apply' to
apply the changes.
```

re: 

>  still need content for real dart fix docs, though...

Do you mean a separate page for `dart fix`?

cc @bwilkerson

Reply:
Yes, I'd like to have a separate page for each of the `dart` commands. It doesn't have to happen in this PR, though.

Reply:
@devoncarew is the old `dartfix` obsolete as of 2.12?

Reply:
yes

Reply:
Thanks, I'll make sure to update /tools

Reply:
I've updated /tools in this PR.